__tests__/app.test.js

```diff
@@ -55,6 +55,7 @@ describe("app.js", () => {
         body: "I find this existence challenging",
         created_at: "2020-07-09T20:11:00.000Z",
         votes: 100,
+        comment_count: 11,
```

Comment:
We should be asserting this new property on a new test, as a general rule we should never go back and update old tests 👍